frontend/src/apis/workOrders/CreateWorkOrder.js

```diff
@@ -10,31 +10,31 @@ var api = axios.create({
 api.interceptors.response.use(async (response) => {
     return await response;
     }, async (error) => {
-        if (error.code == 'ECONNABORTED' || error.response.data.statusCode == '500') {
-            throw new Error('Internal server error. Please try again later.')
+        if (error.code === 'ECONNABORTED' || error.response.data.statusCode == '500') {
+            throw new Error('Internal server error. Please try again later.');
         } else if (error.response && error.response.data.statusCode > 400) {
-            throw new Error('Invalid something.')
+            throw new Error('Invalid entry.');
         } else {
             throw error;
         }
 });
 
-export async function createWorkOrder(propertyId, sector, type, title, cause, service_needed, priority, 
-    description, due_date, price_estimate) {
+export async function createWorkOrder(propertyId, sector, type, title, cause, serviceNeeded, priority, 
+    description, due_date, priceEstimate) {
 
-    body = {
+    var body = {
         sector, 
         type, 
         title, 
         cause, 
-        service_needed: JSON.stringify(service_needed), 
+        serviceNeeded: JSON.stringify(serviceNeeded), 
         priority, 
         description, 
         due_date, 
-        price_estimate: JSON.stringify(price_estimate)
-    }
+        priceEstimate: JSON.stringify(priceEstimate)
```

Comment:
this will fail, API expects underscores and no camel casing for the keys, so if this issue is raised by codacy, you can manually ignore it.